src/chrome/content/rules/ThePirateBay.xml

```diff
@@ -1,50 +1,34 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://kanyebay.com/ => https://kanyebay.com/: (28, 'Connection timed out after 20000 milliseconds')
-Fetch error: http://www.kanyebay.com/ => https://www.kanyebay.com/: (28, 'Connection timed out after 20000 milliseconds')
-
-Other The Pirate Bay rulesets:
-thepiratebay-legacy.xml
-proxybay.one.xml
-thepiratebay-proxylist.org.xml
-thepirateportal.org.xml
-thephoenixbay.com.xml
-proxyportal.org.xml
-tpblist.xyz.xml
-proxybay.pl.xml
-
-www.proxybay.la timed out
-promobay.org nonexist
-www.promobay.org nonexist
-oldpiratebay.org timed out
-www.oldpiratebay.org timed out
+	Other The Pirate Bay rulesets:
+		+ thepiratebay-legacy.xml
+		+ proxybay.one.xml
+		+ thepiratebay-proxylist.org.xml
+		+ thepirateportal.org.xml
+		+ thephoenixbay.com.xml
+		+ proxyportal.org.xml
+		+ tpblist.xyz.xml
+		+ proxybay.pl.xml
+
+	Non-functional hosts
+		Timeout was reached:
+			 - kanyebay.com
+			 - www.kanyebay.com
+			 - oldpiratebay.org
+			 - www.oldpiratebay.org
 -->
-
-<ruleset name="The Pirate Bay" default_off='failed ruleset test'>
-
-	<target host="thepiratebay.se" />
-	<target host="www.thepiratebay.se" />
-	<target host="thepiratebay.org" />
-	<target host="www.thepiratebay.org" />
-	<target host="pirates-forum.org" />
-	<target host="www.pirates-forum.org" />
-	<target host="kanyebay.com" />
-	<target host="www.kanyebay.com" />
-
+<ruleset name="The Pirate Bay">
 	<target host="proxybay.la" />
 	<target host="www.proxybay.la" />
+	<target host="pirates-forum.org" />
+	<target host="www.pirates-forum.org" />
+	<target host="thepiratebay.org" />
+	<target host="www.thepiratebay.org" />
 	<target host="tpb.run" />
 	<target host="www.tpb.run" />
-	
-	<rule from="^http://www\.proxybay\.la/"
-		to="https://proxybay.la/" />
-
-	<target host="thepiratebay-proxylist.org" />
-	<target host="www.thepiratebay-proxylist.org" />
```

Comment:
Those targets have been removed, but they aren't covered by `thepiratebay-proxylist.org.xml` (which is disabled anyway).